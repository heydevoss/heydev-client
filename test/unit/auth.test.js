import { shallowMount } from '@vue/test-utils'
import Auth from '@/pages/auth/_token.vue'

describe('<Auth/> Page', () => {
  const $route = {
    params: {
      token: 'testeteste123'
    }
  }
  const onLogin = jest.fn()
  const wrapper = shallowMount(Auth, {
    mocks: {
      $route,
      $apolloHelpers: {
        onLogin
      },
      $router: {
        replace: jest.fn()
      }
    }
  })

  it('should be a Vue instance and renders correctly', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
    expect(wrapper.element).toMatchSnapshot()
  })
})
