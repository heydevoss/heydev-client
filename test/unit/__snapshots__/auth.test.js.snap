// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Auth/> Page should be a Vue instance and renders correctly 1`] = `<div />`;
